lib/audit/auditor.go

```diff
@@ -392,7 +392,7 @@ func (s *ServerContext) Audit(w http.ResponseWriter, r *http.Request) {
 	// If we can't find the source registry for this image, then reject the
 	// transaction.
 	if string(srcRegistry.Name) == "" {
-		msg := fmt.Sprintf("(%s) TRANSACTION REJECTED: could not determine source registry: %v", s.ID)
+		msg := fmt.Sprintf("(%s) TRANSACTION REJECTED: could not determine source registry: %v", s.ID, gcrPayload)
```

Comment:
Is there any chance of over-logging with gcrPayload?  Sharing something that shouldn't be in the log?

Reply:
No, the  GCR payload only has very sparse details --- the image path, digest, and action (e.g., INSERT, DELETE)